packages/engine/src/simulation/command.rs

```diff
@@ -6,6 +6,7 @@
 use std::{collections::HashSet, sync::Arc};
```

Comment:
It's too far away from the changes, so Github won't let me comment in-line, but the module doc-comments need expanding. (Can you also check the other doc-comments in this module)

Reply:
Good catch, will do!

Reply:
Adjusted the documentation in https://github.com/hashintel/hash/pull/235/commits/891ffc6bc5ae00d6dab30110457dd9890a832489